src/services/infra/InfraService.ts

```diff
@@ -52,6 +54,7 @@ type CreateSiteParams = {
   isEmailLogin: boolean
 }
 
+const ARE_QUEUES_DEPRECIATED = false
```

Comment:
if this is meant for feature flagging, then it should be an env var. this is because setting this to `true` still involves a code deployment to our backend (during which we might as well just change the actual code).

changing it to env vars allows us to deploy the required code without an associated code change (ie, we can deploy first and delete later)

Reply:
Yes that's right. Feature flags modifications shouldn't require a full re-deploy. Hence it shld be in the env vars

Reply:
Sure, modified to use env vars instead here faed0d0c6cd27a6c2a5c9aa02c5c28acbf797d6c

Reply:
shall we impose a standard to prefix all feature flags with a FF_ARE_QUEUES_DEPRECATED?

This will allow us to quickly understand an env var is a feature flag toggle

Reply:
nit: the flag should be something like a verb instead of a question

like `DISABLE_SITE_LAUNCH_QUEUES` or `DEPRECATE_SITE_QUEUES`

Reply:
0b0dbb1e70d6a38aacf079e41cdee38bebe4a9c4